LSP/Example5.md

```diff
@@ -0,0 +1,23 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/angular/angular/blob/main/packages/platform-server/src/domino_adapter.ts#L41
```

Comment:
➕ ➖
If we look at the whole class there are some unexpected behaviour in getCookie method, but exactly makeCurrent method looks good I think.